bitnami/logstash/templates/sts.yaml

```diff
@@ -128,6 +159,28 @@ spec:
           configMap:
             name: {{ include "logstash.configmapName" . }}
         {{- end }}
+        {{- if and .Values.persistence.enabled .Values.persistence.existingClaim }}
+        - name: data
+          persistentVolumeClaim:
+            claimName: {{ .Values.persistence.existingClaim | default (include "common.names.fullname" .) }}
```

Comment:
Why this default? I find it unnecessary since you're already checking 3 lines before that `persistence.existingClaim` is set.

Reply:
Thanks, this helped me fix an issue.

Indeed, I removed the condition because otherwise, it would be necessary to specify the ClaimName when upgrading.